src/utils/sourcemaps/inject.rs

```diff
@@ -121,14 +165,35 @@ pub fn fixup_js_file(js_contents: &mut Vec<u8>, debug_id: DebugId) -> Result<()>
 
     let sourcemap_comment = sourcemap_comment_idx.map(|idx| js_lines.remove(idx));
 
-    for line in js_lines.into_iter() {
-        writeln!(js_contents, "{line}")?;
+    let mut js_lines = js_lines.into_iter().peekable();
+
+    // Write comments and empty lines at the start back to the file
+    while let Some(comment_or_empty) =
+        js_lines.next_if(|line| line.trim().is_empty() || line.trim().starts_with("//"))
+    {
+        writeln!(js_contents, "{comment_or_empty}")?;
     }
 
+    // Write one "use strict" statement back to the file, if there is one
```

Comment:
Did you consult the other pragmas with SDK team? Maybe we can just just append all lines starting with `'` or `"`, just like we do in the check above? (They are all ignored by the parsers anyway)